API/src/microservices/Identity-Management-Service/index.js

```diff
@@ -0,0 +1,41 @@
+const { Users } = require(`../Database`);
+const { Password } = require(`../../utils`);
+
+exports.supervisorLogin = (login) => {
+  return new Promise(async (resolve, reject) => { //eslint-disable-line
+    try {
+
+      let storedSupervisor = await new Users().where({id: 1}).fetch().catch(function (e) {
+        console.log('error in retrieving supervisor information')
+        resolve();
+      });
```

Comment:
Can you explain what's going on here?  Why are you supplying a static id of 1 here?  I guess you're saying the first user is ALWAYS the supervisor?

Reply:
Yes, the supervisor data was entered into the database directly, and is always the first record. 
I do check to see if  both the username and encrypted password match the supervisor login data sent by the form.

I only included the login form with out the register/sign up because that is how I understood the instructions.
I also added a supervisor user into the database table directly, as that is how I understood the instructions. 

Regarding the login for users, I agree its not very logical to have it.

I am sorry I misunderstood the instructions, and I am happy to fix this.
I am thinking of allowing all users to sign up.
I will have 2 buttons in the sign up form. 1) User Sign Up , 2) Supervisor Sign Up 
The difference will be represented in the database in the 'is_supervisor' column as true or false.
Then when the user logs in, assuming his data is correct, he will have a different view depending on whether or not he is supervisor or not.
Would this be a good?